### PR TITLE
[8.x] Adds `Password` rule object documentation 

### PR DESCRIPTION
**Pull request by nunomaduro**
This pull request adds documentation about the `Password` rule object: https://github.com/laravel/framework/pull/36960.

### CODE REVIEW

File: passwords.md [hunk 1]
```diff
@@ -110,12 +110,13 @@ Of course, we need to define a route to actually handle the password reset form
     use Illuminate\Support\Facades\Hash;
     use Illuminate\Support\Facades\Password;
     use Illuminate\Support\Str;
+    use Illuminate\Validation\Rules\Password;
```
Comment (brunogaspar) [inline passwords.md:113]: I know it's documentation, but, this clashes with the facade import above, people will complain lol

Comment (nunomaduro) [reply passwords.md:113]: Ups, nice catch. One second.

Comment (nunomaduro) [reply passwords.md]: fixed

File: validation.md [hunk 2]
```diff
@@ -1412,6 +1413,55 @@ Likewise, you may use the `*` character when specifying [custom validation messa
...
+    Password::min(8)->uncompromised()
+
+Internally, the `Password` rule object uses what is known as a [k-Anonymity](https://en.wikipedia.org/wiki/K-anonymity) model that allows for the password to be looked up on the [haveibeenpwned.com](https://haveibeenpwned.com) service without giving up the user's privacy or security.
+
+By default, if a password appears at least once in a data leak, it will be considered compromised. You can customize this threshold by using the first argument of the `uncompromised` method:
+
+    // Ensure the given password has not been compromised 3 times in the same data leak
+    Password::min(8)->uncompromised(3);
+
+Of course, may chain all the methods specified above:
```
Comment (tomirons) [inline validation.md:1456]: Small typo 😉 
```suggestion
Of course, you may chain all the methods specified above:
```

**Review by tomirons — CHANGES_REQUESTED** [review]

**Pull request merged**